manager/testdata/new-blueprints/read-transform-blueprint.yaml

```diff
@@ -0,0 +1,95 @@
+  labels:
+spec:
+  cluster: thegreendragon
+  appSelector: # Selector of the application that uses this workload
+    clusterName: thegreendragon
+    workloadSelector:
+      matchLabels:
+        app: demoapp
+  steps:
+  - name: read-data
+    assetID: m4d-notebook-sample/paysim-csv
+    templateSpec:
+      chart: ghcr.io/mesh-for-data/m4d-arrow-flight-common-read-conf:0.1.0
+      m4dModule: arrow-flight-common-read-conf # This is the name of the deployed M4DModule definition
+    moduleLabel: "mod: common-arrow-flight-read-service"
+    source:
+      vault:
+        address: http://vault.m4d-system:8200
+        authPath: /v1/auth/kubernetes/login
+        role: module
+        secretPath: /v1/kubernetes-secrets/paysim-csv?namespace=m4d-notebook-sample
+      connection:
+        s3:
+          endpoint: localhost:8001
+          bucket: srcbucket
+          object: data.parq
+    service:
+      name: common-arrow-flight-read-service
+      assetID: m4d-notebook-sample/paysim-csv # Differentiator to original asset for multi user arrow flight service? How to best specify?
+      port: 8080
+      interfaceDetails:
+        protocol: arrow-flight
+        dataformat: arrow
+    transformations: [] # No transformations done in this module
+  - name: transform-step
+    assetID: m4d-notebook-sample/paysim-csv
+    templateSpec:
+      chart: ghcr.io/mesh-for-data/m4d-transform-conf-arrow-flight:0.1.0
+      moduleLabel: "mod: common-transform-arrow-flight"
+    source:
+      step: read-data # the name of step for debugging purposes
+      selector: "mod: common-arrow-flight-read-service" # Selector of the dependency
+      connection:
+        service:
+          name: common-arrow-flight-read-service
+          assetID: m4d-notebook-sample/paysim-csv
+          port: 8080
+          interfaceDetails:
+            protocol: arrow-flight
+            dataformat: arrow
+    service:
+      name: common-transform-arrow-flight
+      assetID: m4d-notebook-sample/paysim-csv
+      port: 8080
+      interfaceDetails:
+        protocol: arrow-flight
+        dataformat: arrow
+    transformations:
+    - action: redact
+      column: blood_group
+  - name: isolation-step1
+    templateSpec: # Configure here or in “reading” module?
+      name: ghcr.io/mesh-for-data/networkpolicy-isolation:0.1.0
+    sourceSelector: "mod: common-read-arrow-flight"
+    moduleSelector: "mod: common-transform-arrow-flight"
+  - name: isolation-step2-to-app
+    templateSpec: # Configure here or in “reading” module?
+      name: ghcr.io/mesh-for-data/networkpolicy-isolation:0.1.0
+    appSelector: "app: demoapp"
+    moduleSelector: "mod: common-transform-arrow-flight"
+status:
+  observedState: Ready
+  observedGeneration: 1
+  steps:
+    - name: read-data
+      status: Ready
+    - name: transform-step
+      status: Ready
+    - name: isolation-step1
+      status: Ready
+    - name: isolation-step2-to-app
+      status: Ready
+  assets:
```

Comment:
Should this be in the blueprint?  A particular asset data path may have modules running in different clusters / blueprints, thus this would not represent its true state.

Reply:
This represents the state of all modules related to this data asset on this cluster correct. This I thought might be good for debugging purposes for developers/admins. (As `Blueprints` are not meant to be viewed by users anyways)